Colours_And_Fonts_index_v1.html

```diff
@@ -75,9 +69,13 @@
 			   <label for="img4"></label>
 			</div>
 		</div>
+	</div>
 			
-			
-			<p><br>As part of the Year 10 programme, students will be participating in four days of Education Outside the Classroom (EOTC). The focus of this programme is on citizenship and social awareness.<br>
+<div class="content">	
+			   <div class="scroll-area">
+				   
+		
+			<div class="para1"><br>As part of the Year 10 programme, students will be participating in four days of Education Outside the Classroom (EOTC). The focus of this programme is on citizenship and social awareness.<br>

```

Comment:
I cahnged <p> to a class so I add add different paragraphs and format them differently as I don't want the text to all be the same it can be more appealing to the end users.